test-vectors.md

```diff
@@ -424,6 +424,38 @@ Verification:
   * `confirmed_transcript_hash = confirmed_transcript_hash_before`
   * `extensions = {}`
 
+## Welcome
+
+Parameters:
+* Ciphersuite
+
+Format:
+
+```text
+{
+  "cipher_suite": /* uint16 */,
+
+  // Chosen by the generator
+  "init_priv": /* hex-encoded serialized HPKE private key */,
+  "signer_pub": /* hex-encoded serialized signature public key */,
+
+  "key_package": /* hex-encoded serialized KeyPackage */,
+  "welcome": /* hex-encoded serialized Welcome */,
```

Comment:
Ping-ponging your suggestion from #82 :-) Should we make these `MLSMessage`, too?

Reply:
Also the KeyPackage!